cookbook/catalog_structure/creating_a_reference_data.rst

```diff
@@ -0,0 +1,232 @@
+How to Create a Reference Data
+==============================
+
+Creating the Reference Data Entity
+----------------------------------
+
+As Akeneo relies heavily on standard tools like Doctrine, creating the entity is
+quite straightforward for any developer with Doctrine experience.
+
+.. note::
+    At the moment, Reference Data can only be stored in ORM. No MongoDB support is provided.
```

Comment:
I would also add that Reference Data offers no CRUD functionnality and that we will still have to deal with custom entity bundle (or whatever external CRUD manager). 

Maybe a little historical point like : 
- 1.1/1.2/1.3 = CustomEntityBundle
 - 1.4 = PIM Native support + CRUD with CustomEntityBundle (or whatever CRUD manager, even external)